src/main/java/games/strategy/engine/data/GameParser.java

```diff
@@ -909,23 +909,8 @@ private void parseProperties(final Node root) throws GameParseException {
         }
       }
     }
-    // add properties for all triplea related maps here:
-    if (!runningList.contains(Constants.AI_BONUS_INCOME_FLAT_RATE)) {
-      data.getProperties()
-          .addEditableProperty(new NumberProperty(Constants.AI_BONUS_INCOME_FLAT_RATE, null, 40, -20, 0));
-    }
-    if (!runningList.contains(Constants.AI_BONUS_INCOME_PERCENTAGE)) {
-      data.getProperties()
-          .addEditableProperty(new NumberProperty(Constants.AI_BONUS_INCOME_PERCENTAGE, null, 200, -100, 0));
-    }
-    if (!runningList.contains(Constants.AI_BONUS_ATTACK)) {
-      data.getProperties()
-          .addEditableProperty(new NumberProperty(Constants.AI_BONUS_ATTACK, null, data.getDiceSides(), 0, 0));
-    }
-    if (!runningList.contains(Constants.AI_BONUS_DEFENSE)) {
-      data.getProperties()
-          .addEditableProperty(new NumberProperty(Constants.AI_BONUS_DEFENSE, null, data.getDiceSides(), 0, 0));
-    }
+    data.getPlayerList().forEach(playerId -> data.getProperties().addEditableProperty(
+        new NumberProperty(playerId.getName() + Constants.BONUS_INCOME_PERCENTAGE, null, 1000, -100, 0)));
```

Comment:
Since we do this (`playerId.getName() + Constants.BONUS_INCOME_PERCENTAGE`) in multiple places, would it be helpful to have a method to encapsulate it?  Something like `Constants.getBonusIncomePercentageFor(PlayerId)`?